plugins/postcss-font-format-keywords/stryker.conf.json

```diff
@@ -0,0 +1,19 @@
+{
+  "$schema": "../../node_modules/@stryker-mutator/core/schema/stryker-schema.json",
+  "mutate": [
+    "src/**/*.ts"
+  ],
+  "buildCommand": "npm run build",
+  "testRunner": "command",
+  "coverageAnalysis": "perTest",
+  "tempDirName": "../../.stryker-tmp",
+  "commandRunner": {
+    "command": "npm run test:tape"
+  },
+  "thresholds": {
+    "high": 100,
+    "low": 100,
+    "break": 100
+  },
+  "inPlace": true
+}
```

Comment:
This file can be removed for now.
There is no complex logic that needs unit tests.